cds_migrator_kit/videos/weblecture_migration/transform/xml_processing/rules/video_lecture.py

```diff
@@ -82,5 +109,124 @@ def event_speakers(self, key, value):
     """Translates event_speakers."""
     name = value.get("p").strip()
     contributor = {"name": name, "role": "Speaker"}
-    # TODO contributor affiliation will be implemented
+    affiliation = value.get("u", "")
```

Comment:
Can we reuse here the `get_contributor` function and enhance only the needed parts e.g role?